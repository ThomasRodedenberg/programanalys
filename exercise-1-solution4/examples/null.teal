fun main() = {
    var x : int := null;
    var y : string := null;
    var z : array[int] := null;
}