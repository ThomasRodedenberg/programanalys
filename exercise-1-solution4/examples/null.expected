A 1 main () -> _3
A 2 x int
A 3 y string
A 4 z array[int]